### PR TITLE
CI: Close PRs not against master

### PR DESCRIPTION
**Pull request by maxime-desroches**
For https://github.com/commaai/openpilot/issues/29845

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
Got an example on your repo of it closing a PR?

**Comment by maxime-desroches** [comment]
https://github.com/maxime-desroches/openpilot/pull/17

File: .github/workflows/labeler.yaml [hunk 1]
```diff
@@ -15,4 +15,17 @@ jobs:
...
       with:
         dot: true
-        configuration-path: .github/labeler.yaml
\ No newline at end of file
+        configuration-path: .github/labeler.yaml
+  pr_branch_check:
+    runs-on: ubuntu-latest
+    if: github.repository == 'commaai/openpilot'
+    steps:
+    - uses: Vankka/pr-target-branch-action@v2
```
Comment (adeebshihadeh) [inline .github/workflows/labeler.yaml:23]: pin this to a specific commit

File: .github/workflows/labeler.yaml [hunk 2]
```diff
@@ -15,4 +15,17 @@ jobs:
...
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        target: /^(?!master$).*/
```
Comment (adeebshihadeh) [inline .github/workflows/labeler.yaml:27]: does this not work?
```suggestion
        target: master
```

**Comment by maxime-desroches** [comment]
Not automatically closed from a fork to master -> https://github.com/maxime-desroches/openpilot/pull/18

Not automatically closed from not a fork to not master -> https://github.com/maxime-desroches/openpilot/pull/19

File: .github/workflows/labeler.yaml (hunk 2 above)
Comment (maxime-desroches) [reply .github/workflows/labeler.yaml:27]: target is the name of the branches that you are not allowed to make a pr against. So in this case we want anything other than "master"

**Comment by adeebshihadeh** [comment]
Congrats on claiming the first bounty!

**Pull request merged**